Kernel/Thread.h

```diff
@@ -1244,6 +1244,9 @@ class Thread
     bool is_promise_violation_pending() const { return m_is_promise_violation_pending; }
     void set_promise_violation_pending(bool value) { m_is_promise_violation_pending = value; }
 
+    bool get_allocation_enabled() const { return m_allocation_enabled; }
```

Comment:
The pattern is to prefix these with `is_` (see examples above). So this should be `is_allocation_enabled()`